packages/expo-cli/src/commands/run/ios/runIos.ts

```diff
@@ -83,6 +90,38 @@ export async function runIosActionAsync(projectRoot: string, options: Options) {
   }
 }
 
+function track(projectRoot: string, exp: ExpoConfig) {
```

Comment:
seems like the only dif from the android `track` command is the platform? If we don't expect custom handling in each case I would lean towards some shared function, but I think this is fine as well 👍